### PR TITLE
Remove id webapp refs

### PR DESCRIPTION
**Pull request by markjamesbutler**
We no longer use the old identity webapp for social sign-in therefore removing the links to the old webapp.

### CODE REVIEW

**Comment by rich-nguyen** [comment]
Does this mean you can remove `IdentitySocialOAuthSwitch` from `FeatureSwitches.scala`? would be good to get rid of any unused switches. :+1:

File: identity/app/views/fragments/socialSignin.scala.html [hunk 1]
```diff
@@ -20,11 +16,7 @@
     <li class="social-signin__item">
         <a class="social-signin__action social-signin__action--gplus"
             data-link-name="Register/Sign in with Google" data-test-id="google-sign-in"
-            href="@if(IdentitySocialOAuthSwitch.isSwitchedOn) {
```
Comment (jennysivapalan) [inline identity/app/views/fragments/socialSignin.scala.html]: You can also remove the `IdentitySocialOAuthSwitch` switch.

**Comment by jennysivapalan** [comment]
@rich-nguyen jinx!

**Comment by johnduffell** [comment]
what's happening with this one?

File: identity/app/views/fragments/socialSignin.scala.html (hunk 1 above)
Comment (markjamesbutler) [inline identity/app/views/fragments/socialSignin.scala.html]: I will do this in a subsequent PR.

**Pull request merged**